src/axom/primal/geometry/KnotVector.hpp

```diff
@@ -546,25 +546,25 @@ class KnotVector
   {
     SLIC_ASSERT(isValidSpan(span, t));
 
-    const int m_deg = getDegree();
+    const int deg = getDegree();
 
     axom::Array<axom::Array<T>> ders(n + 1);
 
     axom::Array<axom::Array<T>> ndu(m_deg + 1), a(2);
     axom::Array<T> left(m_deg + 1), right(m_deg + 1);
     for(int j = 0; j <= m_deg; j++)
```

Comment:
If `deg` is local, then these should probably used `deg` instead of `m_deg`

Reply:
Since `deg` and `m_deg` are the same, I used `m_deg` throughout, as is done in other methods.  Removing the assignment as @jcs15c suggested.

Reply:
Thanks!